packages/aws-sdk-signers/src/aws_sdk_signers/__init__.py

```diff
@@ -16,7 +16,7 @@
 )
 
 __license__ = "Apache-2.0"
-__version__ = "0.1.0"
+__version__ = "0.2.0"
```

Comment:
Why does signers need a minor version bump? This is a leaf note in the dependency tree of the SDKs and `smithy-aws-core`. There are also no new changes being added to this package. Personally, I don't think we should be bumping this.